.coveragerc

```diff
@@ -40,6 +40,7 @@ omit =
     homeassistant/components/*/mysensors.py
 
     homeassistant/components/binary_sensor/arest.py
+    homeassistant/components/binary_sensor/nest.py
```

Comment:
Make sure you add the nest component too. Could you follow a similar pattern to `mysensors`?